src/desktop_pixmap.cpp

```diff
@@ -111,15 +126,18 @@ desktop_pixmap::save_screen (pixmap_format pixmap_format)
                      desktop_pixmap::gc_, this->width, this->height, 0,
                      y_exception, 0, drawable::screen_->root_depth,
                      this->length, this->image_ptr);
+
       i++;
+
       // Not freeing gi_reply causes memory leak,
       // as xcb_get_image always allocates new space
       // This deallocates saved image on x server
       delete[] this->gi_reply;
```

Comment:
Replace all `delete[]` with `free()` for `this->gi_reply` as this may break compatibility with FreeBSD